### ISSUE TITLE
archive.is: "no result" because of Google reCaptcha

### ISSUE THREAD

**Issue by HLFH** [searx/searx#2022]
As seen in searx engine stats: archive.is engine no longer works and does not provide any result: https://searx.space/#

Shall we remove this engine?

**Comment by return42** [comment]
We should open an issue if our scrapper is buggy and gives zero results. ...If this is the case, tag this issue as _bug_. If the origin search engine does no longer exists, we should remove the engine from our tree.

**Comment by unixfox** [comment]
> We should open an issue if our scrapper is buggy and gives zero results. ...If this is the case, tag this issue as _bug_. If the origin search engine does no longer exists, we should remove the engine from our tree.

That's true. An engine that no longer work doesn't mean we need to remove it but fix it instead.

**Comment by HLFH** [comment]
@unixfox @return42 Ok, understood. Thanks.

**Comment by HLFH** [comment]
[archive.is](https://en.wikipedia.org/wiki/Archive.today) is often using Google ReCaptcha. That might be why, it is not working.

Related to: https://github.com/asciimoo/searx/issues/729, https://github.com/asciimoo/searx/issues/1812

```html
         <div id="g-recaptcha"></div>
         var widgetId = grecaptcha.render('g-recaptcha', {
         xhr.open('POST', "/cdn-cgi/l/chk_captcha");
         xhr.onload = function() { if (xhr.status == 200) window.location.reload(true); else grecaptcha.reset(widgetId); }
         xhr.onerror = function() { grecaptcha.reset(widgetId); }
         <script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" async defer></script>
```

In searx debug mode, you would get:

> juil. 12 19:23:44 arch-server.dhautefeuille.eu uwsgi[1060259]: DEBUG:urllib3.connectionpool:https://archive.is:443 "GET /time HTTP/1.1" 503 54547

@unixfox

**Comment by unixfox** [comment]
~~@HLFH is it working fine in a local searx instance?~~
Just tested it is not.
